example/src/main/java/io/scalecube/benchmarks/example/ExampleServicesBenchmarksState.java

```diff
@@ -0,0 +1,75 @@
+package io.scalecube.benchmarks.example;
+
+import io.scalecube.benchmarks.BenchmarksSettings;
+import io.scalecube.benchmarks.BenchmarksState;
+import io.scalecube.services.Microservices;
+import io.scalecube.services.ServiceCall;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+
+public class ExampleServicesBenchmarksState extends BenchmarksState<ExampleServicesBenchmarksState> {
```

Comment:
Remove dependency on `scalecube-services` ; just put a simplier examples w/o services and cluster.